### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by mukhammart** [spacedriveapp/spacedrive#1661] [Feature Request]
I would love to see some preferable languages in app, I'm a technical writer and would like to translate the UI of app into Russian, whereas I'm Uzbek)

**Comment by brxken128** [comment]
I explained a little [here](https://github.com/spacedriveapp/spacedrive/discussions/1668#discussioncomment-7369135), but currently translations would be quite hard to provide given how quickly/rapidly major components of the can app change every single day.

We aim to keep Rust as the functional "powerhouse" behind the application, and the end goal is to have it provide only raw data to the frontend (such as the progress of a certain job), so that localisation and multiple languages can be supported and updated with relative ease.

**Comment by JiagengDing** [comment]
I really like this project a lot! 

If there any plan to support more languages, please contact me; I'd be willing to contribute Simplified Chinese translation files.
